.github/workflows/build-macos.yml

```diff
@@ -65,7 +69,7 @@ jobs:
         # Codecov no longer parses gcov files automatically
         run: |
           rm -rf build
-          meson setup build --prefix=$HOME/.local -Db_coverage=true --buildtype=debug ${{ matrix.build_opts }}
+          meson setup build --prefix=$HOME/.local -Db_coverage=true --buildtype=debug
```

Comment:
This still needs to take `$BUILD_OPTS` so we include an LTO'd coverage build built with all static libraries in the matrix

Reply:
Lto is on by default, it's just not statically linked which should not matter for coverage? Happy to add it anyway, let me know

Reply:
It's really to cover all our bases as LTO+static is the tricky combination for macOS due to how it defines symbol tables, so we'd like it to just give us the most complete matrix we can for the number of builds.

Reply:
Oki